devices/ikea.js

```diff
@@ -585,4 +585,13 @@ module.exports = [
         description: 'TRADFRI LED bulb E14 WW clear 250 lumen, dimmable',
         extend: tradfriExtend.light_onoff_brightness(),
     },
+    {
+        zigbeeModel: ['BADRING Water Leakage Sensor'],
+        model: 'BADRING Water Leakage Sensor',
```

Comment:
Is there any model number written on the box or manual? Something like EXXXX?

Reply:
Yes. E2202!

Reply:
Great, can you update the PR and rebase your branch? It has a merge conflict at the moment.